.github/workflows/ci.yaml

```diff
@@ -358,18 +358,23 @@ jobs:
         run: mix setup
       - name: Run trento detached
         run: mix phx.server &
-      - name: Get Photofinish for e2e testing
-        run: |
-          set -x
-          wget -P /tmp https://github.com/trento-project/photofinish/releases/download/v1.1.0/photofinish
-          chmod +x /tmp/photofinish
+      - name: Install photofinish
+        uses: jaxxstorm/action-install-gh-release@v1.5.0
+        with:
+          repo: trento-project/photofinish
+          tag: v1.1.2
+          cache: enable
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Give executable permissions to photofinish
+        run: chmod +x $(whereis photofinish | cut -d" " -f2)
```

Comment:
Maybe we could replace `whereis` by `which`. This 2nd returns the binary path

Reply:
@arbulu89 I had already tried but from what I could see, `which` returns only results that already have the correct permissions:

```
[rtorrero@galileo ~]$ which photofinish
which: no photofinish in (/home/rtorrero/.mix/escripts:/home/rtorrero/go/bin:/home/rtorrero/.local/bin...)

[rtorrero@galileo ~]$ whereis photofinish
photofinish: /usr/bin/photofinish

[rtorrero@galileo ~]$ sudo chmod +x /usr/bin/photofinish 
[rtorrero@galileo ~]$ which photofinish
/usr/bin/photofinish

```

Reply:
So you could use `which` in the 2nd call XD